src/scripts/ordo_subsets.py

```diff
@@ -0,0 +1,84 @@
+"""Creates a ROBOT template to update mondo terms with the correct ORDO subset annotation."""
+from argparse import ArgumentParser
+
+import pandas as pd
+
+from utils import remove_angle_brackets
+
+
+SUBSET_MAP = {
+    'Orphanet:557493': 'http://purl.obolibrary.org/obo/mondo#ordo_disease',  # disorder
+    'Orphanet:557492': 'http://purl.obolibrary.org/obo/mondo#ordo_group_of_disorders',  # group of disorders
+    'Orphanet:557494': 'http://purl.obolibrary.org/obo/mondo#ordo_clinical_subtype',  # subtype of a disorder
+}
+ROBOT_TEMPLATE_HEADER = {
+    'mondo_id': 'ID',
+    'subset': 'A oboInOwl:inSubset',
+    'ordo_id': '>A oboInOwl:source',
+}
+
+
```

Comment:
Investigate why these four classes without mappings ended up in the subset:

| Mondo ID      | label                                                                                                                   |
|---------------|-------------------------------------------------------------------------------------------------------------------------|
| MONDO:0011965 | familial temporal lobe epilepsy 2                                                                                       |
| MONDO:0009349 | holoprosencephaly 1                                                                                                     |
| MONDO:0100340 | Friedreich ataxia 1                                                                                                     |
| MONDO:0024525 | Fanconi renotubular syndrome 1                                                                                          |

Mandy using and outdate mondo.sssom.tsv?

Reply:
That's probably it. I forgot and didn't run with `-B` or delete `mondo.sssom.tsv` first.

Reply:
None of these 4 are in the gSheet.

Reply:
I deleted them from the gsheet for the pipeline processing

Reply:
@matentzn Looked into this. I got a fresh copy of `mondo.sssom.tsv`, and these 4 classes do indeed still have mappings. So there seems to be no issue w/ the code in this PR; they seem like they should show up in our subset results.

Just FYI if you want to think through this some more, I found the original comment where these 4 classes were discussed; idk if it might help to revisit/reread: [comment](https://github.com/monarch-initiative/mondo/pull/7681#issuecomment-2098193144)

Reply:
Did you get a fresh copy of mondo.sssom.tsv, or use the mondo-ingest pipeline to generate a fresh copy? I hope the latter! If the mondo.sssom.tsv mapping file still contains "exact matches" to these four terms, can you check if they also exist in mondo-edit.obo? Because I determined they dont, but I may have made a mistake somewhere along the way.

Reply:
@matentzn My bad; I made a boo-boo. I did run the goal in `mondo-ingest`, but not correctly.
@twhetzel When we went over this, I skipped through it too quickly. Even though the goal for `mondo.sssom.tsv` does do `cp $(TMPDIR)/mondo/src/ontology/mappings/mondo.sssom.tsv $@`,the prerequisite for this goal does indeed run `make mondo.owl mappings`. During our checking, it was not sufficient to simply `rm tmp/mondo.sssom.tsv; make tmp/mondo.sssom.tsv`. We needed to just do `make tmp/mondo.sssom.tsv -B` to rerun the pipeline.

In the past, I've sometimes run this incorrectly, and sometimes have run it correctly. Basically I often don't use `-B` for a lot of goals because it often runs a long pipeline that is not actually necessary at the time. 

I'm re-running it now but it takes a while. But I expect that indeed, these 4 classes will not show up after I'm done. I don't see any ORDO xrefs on them in the `mondo.sssom.tsv` in my `mondo` repo, nor in `mondo-edit.obo`.

---

@matentzn suggestion:
In order to make this better dummy-proof UX for me / in general, what do you think about this change to the goals around `tmp/mondo_repo_built`?

<details><summary>Current goal setup</summary>
<p>

```make
tmp/mondo_repo_built:
	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
		rm -rf ./mondo/ &&\
		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
		cd mondo/src/ontology &&\
		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
		cd ../../../../ &&\
		touch $@; fi

$(TMPDIR)/mondo.sssom.tsv: $(TMPDIR)/mondo_repo_built
	cp $(TMPDIR)/mondo/src/ontology/mappings/mondo.sssom.tsv $@

$(TMPDIR)/mondo.owl: $(TMPDIR)/mondo_repo_built
	cp $(TMPDIR)/mondo/src/ontology/mondo.owl $@
```

</p>
</details> 

<details><summary>Proposed changes</summary>
<p>

```make
$(TMPDIR)/mondo.sssom.tsv:
	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
		rm -rf ./mondo/ &&\
		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
		cd mondo/src/ontology &&\
		make mappings -B MIR=false IMP=false MIR=false &&; fi
	cp $(TMPDIR)/mondo/src/ontology/mappings/mondo.sssom.tsv $@

$(TMPDIR)/mondo.owl:
	if [ $(USE_MONDO_RELEASE) = true ]; then wget http://purl.obolibrary.org/obo/mondo.owl -O $@; else cd $(TMPDIR) &&\
		rm -rf ./mondo/ &&\
		git clone --depth 1 https://github.com/monarch-initiative/mondo &&\
		cd mondo/src/ontology &&\
		make mondo.owl -B MIR=false IMP=false MIR=false &&; fi
	cp $(TMPDIR)/mondo/src/ontology/mondo.owl $@
```

</p>
</details>

Reply:
These suggestions would break the point of `make` caching though, which in turn would make running the pipeline much slower if you have to run it repeatedly. My preferred way would be to solve this via SOP:

If you want to ensure the latest version of the mondo repo during your development process, you run:

```
rm tmp/mondo_repo_built
```

Then continue.

Reply:
Ah, you are right. I guess I just have to remember this. I just don't like having to remember a lot of little things. But I suppose I don't see a way around it either. 

In that case if I am just on my own just trying to get mondo.sssom.tsv, I think -B will have the same effect.